File: Scala Kompics/src/se/sics/kompics/scala/performancetest/kompics/kvstore/Executor.scala
package se.sics.kompics.scala.performancetest.kompics.kvstore

import se.sics.kompics.performancetest.scenario.{Topology, Scenario, Profiler}
import se.sics.kompics.Kompics
import java.net.InetAddress

class Executor extends se.sics.kompics.performancetest.scenario.Executor {
	
	override def run(t: Topology, s: Scenario, p: Profiler, ip: InetAddress) = {
		Application.profiler = p;
		Application.topology = t;
		Application.scenario = s;
		Application.ip = ip;
		Kompics.createAndStart(classOf[Application], 12);
	}
	
}

File: Java Kompics/src/se/kth/ii2202/experiments/nnar/Executor.java
package se.kth.ii2202.experiments.nnar;

import java.net.InetAddress;
import se.sics.kompics.Kompics;
import se.sics.kompics.performancetest.scenario.Profiler;
import se.sics.kompics.performancetest.scenario.Scenario;
import se.sics.kompics.performancetest.scenario.Topology;

/**
 *
 * @author carbone
 */
public class Executor implements se.sics.kompics.performancetest.scenario.Executor {

    @Override
    public void run(Topology t, Scenario s, Profiler p, InetAddress ip) {
        Application.profiler = p;
        Application.topology = t;
        Application.scenario = s;
        Application.ip = ip;
        Kompics.createAndStart(Application.class, 12);
    }
}
